apps/web/src/components/custom/theme-provider.tsx

```diff
@@ -0,0 +1,11 @@
+"use client"
+
+import * as React from "react"
+import { ThemeProvider as NextThemesProvider } from "next-themes"
```

Comment:
```suggestion
import { ThemeProvider as NextThemesProvider } from "next-themes"
import type { ComponentProps ] from "react";
```

ALWAYS import your time never use global thing